esx_service/cli/vmdkops_admin.py

```diff
@@ -255,6 +255,20 @@ def commands():
         'status': {
             'func': status,
             'help': 'Show the status of the vmdk_ops service'
+        },
+        'edit': {
```

Comment:
I agree. `update` is also a possibility. `set` is used in the role commands, while `update` is used in the policy command.